### PR TITLE
bump circl.yml to 9.1

### PR DESCRIPTION
**Pull request by yaron-idan**
Trying to close issue #6781, couldn't run CircleCI on my fork since MacOs plan requires billing, so using the PR to test this.

### CODE REVIEW

**Comment by dnkoutso** [comment]
LGTM if it passes. /cc @DanToml

**Comment by yaron-idan** [comment]
@dnkoutso Thanks! This made me wonder though, are you still using Circle? It seems only Travis and Danger are configured for status checks, so perhaps Circle could be ommited?
If you do plan on using it, maybe moving on Circle 2.0 is a good step, I can create an issue&PR if this seems valuable.

**Comment by DanToml** [comment]
@yaron-idan Circle is used mostly for the examples rn - because we were waiting for workflows, I'd be interested in seeing that PR.

**Comment by yaron-idan** [comment]
Cool, I'll probably need access to a Circle account that supports macOS, can you give me access to such an account? For me it's only possible as a paid subscriber but I figure maybe CocoaPods get some open source deal.

**Comment by DanToml** [comment]
@yaron-idan I just enabled OSS macOS builds on your account.. It looks like there's sometimes a bug when running forked prs when someone has followed a fork in circle

**Comment by yaron-idan** [comment]
@DanToml That works, thanks!

**Comment by dnkoutso** [comment]
do we want to land this?

**Comment by yaron-idan** [comment]
@dnkoutso I actually tested my change after being granted access to Circle and it seems some more work has to be done before moving to xcode 9 will be possible. I've started with adding build settings for swift version to examples/AFNetworking Example since Circle was failing on lack of those and then bumped into more errors in AlamoFire Example which seems like it could use a bump up in it's version. 
To sum it up - I find myself a bit over my head, if there's a community you can point at for guidance I'll be happy to seek help there but there seems to be a good amount of work to be put in there to be ready for xcode 9.
